_episodes/12-bonus.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Bonus Episode"
+teaching: 40
+exercises: 0
+questions:
+- How to build Docker container for python packages?
+- How to share Docker images?
+objectives:
+- To be able to build a Docker container and share it via GitHub packages
+keypoints:
+-  Python packages can be installed in Docker images along with ubuntu packages.
+-  It is possible to publish and share Docker images over github packages.
+---
+
+> ## Prerequisites
+> For this lesson, you will need,
+> * Knowledge of Git [SW Carpentry Git-Novice Lesson](https://swcarpentry.github.io/git-novice/)
+> * Knowledge of GitHub CI/CD [HSF Github CI/CD Lesson](https://hsf-training.github.io/hsf-training-cicd-github/)
+{: .prereq}
+
+## Docker Container for python packages
+
+Python packages can be installed using a Docker image. The following example illustrates how to write a Dockerfile for building an image containing python packages.
+
+```text
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update \
+ && apt-get install -yq \
```

Comment:
I don't understand this line: Isn't it missing the package to-be-installed?

Reply:
that was an unnecessary repetition. I have removed it. Thanks.